src/main/java/org/fedoraproject/javapackages/validator/Logger.java

```diff
@@ -1,52 +1,22 @@
 package org.fedoraproject.javapackages.validator;
 
-import java.io.OutputStream;
 import java.io.PrintStream;
-import java.nio.charset.StandardCharsets;
-import java.text.MessageFormat;
 import java.util.EnumMap;
 
-import org.fedoraproject.javapackages.validator.TextDecorator.Decoration;
-
 public class Logger {
```

Comment:
I recommend using a standard logging API such as [SLF4J](https://www.slf4j.org/) instead of maintaining this custom logger.
IMHO text decoration feature is not useful and can be dropped.